openlibrary/plugins/upstream/addtag.py

```diff
@@ -62,6 +66,10 @@ def POST(self):
             raise web.unauthorized(message='Permission denied to add tags')
 
         i = utils.unflatten(i)
+
+        if not i.tag_name or not i.tag_type:
```

Comment:
Will DRY this to use `validate_tag` when work begins on #9784.  Today, the "Tag Name" input is identified differently in both forms (`name` in one; `tag_name` in the other).